build.psm1

```diff
@@ -2069,9 +2079,10 @@ function New-MSIPackage
         [ValidateNotNullOrEmpty()]
         [string] $ProductVersion,
 
-        # Product Guid needs to change for every version to support SxS install
+        # Product Code needs to change for every version to support SxS install
```

Comment:
Does this comment make sense any more?

`The ProductCode property is a unique identifier for the particular product release`

from https://msdn.microsoft.com/en-us/library/windows/desktop/aa370854(v=vs.85).aspx